docs-2.0/nebula-operator/4.connect-to-nebula-graph-service.md

```diff
@@ -0,0 +1,142 @@
+# Connect to Nebula Graph databases with Nebular Operator
+
+After creating a Nebula Graph cluster with Nebula Operator on Kubernetes, you can connect to Nebula Graph databases from within the cluster and outside the cluster.
+
+## Prerequisites
+
+Create a Nebula Graph cluster with Nebula Operator on Kubernetes. For more information, see [Deploy Nebula Graph clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Deploy Nebula Graph clusters with Kubectl](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
```

Comment:
Please double-check the links.
And maybe here is `Deploy Nebula Graph clusters with Kubectl` or `Deploy Nebula Graph clusters with Helm`?

Reply:
yup you're absolutely rith